VideoLocker/src/main/java/org/edx/mobile/view/Router.java

```diff
@@ -34,10 +34,9 @@
     public static final String EXTRA_BUNDLE = "bundle";
     public static final String EXTRA_COURSE_ID = "course_id";
     public static final String EXTRA_COURSE_DETAIL = "course_detail";
-    public static final String EXTRA_ENROLLMENT = "enrollment";
+    public static final String EXTRA_ENROLLMENT_COURSE_DATA = "enrollment_course_data";
```

Comment:
I think the 'enrollment' part is also redundant information here, and this can be simplified to just `EXTRA_COURSE_DATA` and `"course_data"`.